src/components/ContactList.tsx

```diff
@@ -1,7 +1,18 @@
 import { Link } from "react-router-dom";
+import { useState, useEffect } from "react";
 
 const ContactList = () => {
+    const [contacts, setContacts] = useState(null);
+    
+    const fetchContacts = async () => {
+        const response = await fetch('http://localhost:3000/contacts');
+        const contactsData = await response.json();
+        setContacts(contactsData);
```

Comment:
You're not yet rendering this data, so you may want to confirm you're getting what you expect - either by rendering it, or console-logging while it's in development to confirm it is what you expected